frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -130,7 +136,17 @@ function _BaseBucketPickerPopover({
           {hasArrowIcon && !hasChevronDown && (
             <TriggerIcon name="chevronright" />
           )}
-          {hasChevronDown && <ChevronDown name="chevrondown" />}
+          {/*
+            isEditing controls "selected" state of the item, so if a row is selected, we want to show icon
+            otherwise we show chevron down icon only when we hover over a row, to control this behavior
+            we pass or not pass chevronDown class, which hides this icon by default
+          */}
+          {hasChevronDown && (
+            <ChevronDown
+              className={isEditing ? undefined : classes.chevronDown}
```

Comment:
I believe this condition should live in `QueryColumnPicker.tsx` since that's the component which "knows" that this class makes things visible/hidden.